### ISSUE TITLE
Need to add packages to Suggests:

### ISSUE THREAD

**Issue by kbenoit** [quanteda/spacyr#122]
Got this from KH:

> From: Kurt Hornik <Kurt.Hornik@R-project.org>
> Subject: CRAN package spacyr
> Date: 18 June 2018 at 14:19:05 BST
> To: <kbenoit@lse.ac.uk>
> Cc: <CRAN@R-project.org>
> Reply-To: <CRAN@R-project.org>
> 
> Dear maintainer,
> 
> Please see the problems shown on
> <https://cran.r-project.org/web/checks/check_results_spacyr.html>.
> 
> Please correct before 2018-07-02 to safely retain your package on CRAN.
> 
> Best,
> -k
> 

```
Version: 0.9.9 
Check: for unstated dependencies in ‘tests’ 
Result: WARN 
    '::' or ':::' import not declared from: ‘quanteda’ 
Flavors: r-devel-linux-x86_64-debian-clang, r-devel-linux-x86_64-debian-gcc, r-devel-linux-x86_64-fedora-clang, r-devel-linux-x86_64-fedora-gcc
```

**Comment by amatsuo** [comment]
So now they ask us to include packages only in tests?

**Comment by kbenoit** [comment]
Those packages already needed to be in Suggests:, but **quanteda** was already.  I think we just needed to load the package, rather than only use the `quanteda::function()` calls in the tests.

**Comment by kbenoit** [comment]
@amatsuo I fixed this in 2dc067c but please try it on your Linux system, with the `--as-cran` flag set. (And for R 3.5!)

**Comment by amatsuo** [comment]
It's passing on my linux virtual machine.
